.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Contract Development
+
+# Pushes to long living branches and all PRs
+on:
+  push:
+    branches:
+      - master
+      - 0.[0-9]+
+  pull_request:
```

Comment:
I would just reverse the order of `pull` and `pull_request`, to match the config in the master branch.

Reply:
I took this example from a different project, which changed the order. On master it is `push`, `pull_request` as well now (after merging 0.12 -> master).